packages/toolbox/README.md

```diff
@@ -67,7 +67,7 @@ It will suggest pairs of currencies:
 
 - With a Chainlink price feed oracle (according to [a cached JSON](https://cl-docs-addresses.web.app/addresses.json]))
 - If they exist in Currency Manager (cf. [../currency/src/erc20/networks]())
-- If they are not already added to the Chainlink Aggregation Path contract, as reported by the Price Aggregators subgraph ([Example for BSC](https://thegraph.com/hosted-service/subgraph/requestnetwork/price-aggregators-bsc))
+- If they are not already added to the Chainlink Aggregation Path contract, as reported by the Price Aggregators subgraph ([Example for BSC](https://thegraph.com/explorer/subgraphs/H4knum5YFFtgpWjYDV6xqjkPZprtm7S9jv4UsdnTS1hD?view=Overview&chain=arbitrum-one))
```

Comment:
This is the Mainnet URL. Please either make the text say "Mainnet" or change to the BSC URL.